services/pama-imaging.js

```diff
@@ -109,17 +139,48 @@ function getSystemActions(entries, selections) {
   ));
 }
 
-function getCards() {
+function getCards(entries, actions) {
+  const reasonCodes = new Set(entries
+    .map(x => x.resource.reasonCode)
+    .flat()
+    .map(x => x.coding.map(y => y.code))
+    .flat());
+  const cards = Object.entries(recommendations)
+    .filter(x => reasonCodes.has(x[0]))
+    .map(x => x[1])
+    .flat();
+//  if (cards.length === 0) return []; // XXX
+  const recommendedActions = new Set(cards
+    .map(x => x.suggestions.actions
```

Comment:
`card.suggestions` is an array; it shouldn't have an `actions` property.


From [the spec](https://cds-hooks.org/specification/1.0/#card-attributes):
-- | -- | -- | --
suggestions | OPTIONAL | array of Suggestions | Allows a service to suggest a set of changes in the context of the current activity (e.g. changing the dose of the medication currently being prescribed, for the medication-prescribe activity). If suggestions are present, selectionBehavior MUST also be provided.
-- | -- | -- | --

Reply:
Good catch, thanks.

Reply:
Done.